templates/deployment_workflows.yaml

```diff
@@ -64,7 +64,13 @@ spec:
           - name: DBCONNECTOR_POSTGRES_POOL_MAX_SIZE
             value: "100"
           - name: DBCONNECTOR_QUERY_TIMEOUT_MS
+          {{- if .Values.workflows.dbConnectorTimeout }}
+            value: {{ .Values.workflows.dbConnectorTimeout | quote}}
+          {{- else if .Values.config.dbConnectorTimeout }}
+            value: {{ .Values.config.dbConnectorTimeout | quote}}
+          {{- else }}
```

Comment:
don't love this, but keeps backcompat while making future deployments have saner default